src/test/java/seedu/trackbeau/testutil/PersonBuilder.java

```diff
@@ -35,7 +43,11 @@ public PersonBuilder() {
         phone = new Phone(DEFAULT_PHONE);
         email = new Email(DEFAULT_EMAIL);
         address = new Address(DEFAULT_ADDRESS);
-        tags = new HashSet<>();
+        skinType = new SkinType(DEFAULT_SKIN_TYPE);
```

Comment:
Do we need to rename this class to CustomerBuilder? Or can this class be a superclass for both Customer and Staff?